src/index.js

```diff
@@ -94,6 +94,7 @@ window.wikiaJWPlayer = function (elementId, options, callback) {
 
 		wikiaJWPlayerReplaceIcons(playerInstance);
 		wikiaJWPlayerEvents(playerInstance, options.autoplay.enabled, logger);
+		options.related && wikiaJWPlayerRelatedVideoSound(playerInstance);
```

Comment:
IMO `if` is more readable. 
And code probably will be minified to similar statement.